packages/tools/devtools/devtools-view/package.json

```diff
@@ -19,7 +19,7 @@
 		"build:compile": "fluid-build . --task compile",
 		"build:docs": "api-extractor run --local && copyfiles -u 1 ./_api-extractor-temp/doc-models/* ../../../../_api-extractor-temp/",
 		"ci:build:docs": "api-extractor run && copyfiles -u 1 ./_api-extractor-temp/doc-models/* ../../../../_api-extractor-temp/",
-		"clean": "rimraf _api-extractor-temp coverage dist nyc *.tsbuildinfo *.build.log",
+		"clean": "rimraf '_api-extractor-temp' 'coverage' 'dist' 'nyc' '*.tsbuildinfo' '*.build.log' --glob",
```

Comment:
The patterns that have `*` work natively in Unix systems because most shells interpret them as globbing patterns and expand them, but in Windows that doesn't happen. Besides using the `--glob` flag to explicitly make the behavior consistent, I think Windows wants the arguments quoted to work correctly.